Tokens/0x35274271db5193160A02b81826C9272eE01F5B9C/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "WRNA",
+  "website": "https://genechain.io",
```

Comment:
This should not be GeneChain. WRNA is issued by Novaswap.

Reply:
It seems 0xb7a72452d0ce44683C7492289a48F315865cb4B3 and 0x35274271db5193160A02b81826C9272eE01F5B9C are both available on genechain mainnet and testnet. But they are shown as tokens on testnet only.
If they are supposed to be used on testnet, it might be a good idea to mark them as testnet tokens in name and symbol.

Reply:
Sorry. I made a mistake. 0xb7a72452d0ce44683C7492289a48F315865cb4B3 and 0x35274271db5193160A02b81826C9272eE01F5B9C are not created on mainnet.